RecoLocalCalo/HGCalRecProducers/interface/HGCalTilesConstants.h

```diff
@@ -14,14 +14,24 @@ namespace hgcaltilesconstants {
                                                                   : static_cast<int32_t>(num) + ((num > 0) ? 1 : 0);
   }
 
+  constexpr float tileSize = 5.f;
   constexpr float minX = -285.f;
   constexpr float maxX = 285.f;
   constexpr float minY = -285.f;
   constexpr float maxY = 285.f;
-  constexpr float tileSize = 5.f;
   constexpr int nColumns = hgcaltilesconstants::ceil((maxX - minX) / tileSize);
   constexpr int nRows = hgcaltilesconstants::ceil((maxY - minY) / tileSize);
+  constexpr float tileSizeEtaPhi = 0.15f;
+  constexpr float minEta = -3.f;
+  constexpr float maxEta = 3.f;
+  constexpr float minPhi =
+      -3.3f;  //To properly construct search box for cells in phi=[-3.15,-3.], cells in phi=[3.,3.15] are copied to the first bin
+  constexpr float maxPhi =
+      3.3f;  //To properly construct search box for cells in phi=[3.,3.15], cells in phi=[-3.15,-3.] are copied to the last bin
```

Comment:
perhaps this comment can be combined and even placed above the minPhi,maxPhi. This way also the assignment is not going to be broken by the clang format.
The alternatives are to shorten the text or split it in two lines with one line inline and the last aligned to the upper one.

Reply:
Done.